### DISCUSSION TITLE
ERR_FILE_NOT_FOUND error google docs fonts

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3719] [Q&A]
Hi,

While opening the project page we are getting below error. How to solve this?

GET filesystem:https://docs.google.com/persistent/docs/fonts/KFOmCnqEu92Fr1Me4A.woff2 net::ERR_FILE_NOT_FOUND
viewform:1          GET filesystem:https://docs.google.com/persistent/docs/fonts/KFOkCnqEu92Fr1Mu52xK.woff2 net::ERR_FILE_NOT_FOUND
viewform:1          GET filesystem:https://docs.google.com/persistent/docs/fonts/KFOlCnqEu92Fr1MmWUlvBg.woff2 net::ERR_FILE_NOT_FOUND
viewform:1          GET filesystem:https://docs.google.com/persistent/docs/fonts/KFOjCnqEu92Fr1Mu51TzBhc4.woff2 net::ERR_FILE_NOT_FOUND
 
Thanks and Regards,
SumathyVenugopal

**Comment by o1lab** [comment]
Looks like an issue with loading google form.

Are you running this in an environment which blocks google or something ?

@mertmit

**Comment by MGW-SumathyV** [comment]
This error is not occuring in Edge but occuring in Chrome. Our environment is not blocking google
